### PR TITLE
Add templates to deploy tooling on OpenShift

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds:
- pbench-agent and collectd templates.
- a readme with instructions about deploying the tooling.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@openshift/svt

File: openshift_templates/collectd-config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collectd-config
+data:
+  host: " "
```
Comment (mbruzek) [inline openshift_templates/collectd-config.yml]: Why are these one space? Could they be empty string?

Comment (chaitanyaenr) [reply openshift_templates/collectd-config.yml]: Yes, they can be a empty string. Fixed.

**Comment by mbruzek** [comment]
The readme is well formed markdown and makes sense. The templates look fine to me.
/lgtm

**Comment by chaitanyaenr** [comment]
@openshift/svt ready for merge?

**Comment by mffiedler** [comment]
@chaitanyaenr @mbruzek If we're going to create a new toplevel folder in the repo I'd prefer to create subfolders to organize the content underneath.   Otherwise it will just become a dumping ground.   Please add an additional level of dir structure for your content - maybe "performance_monitoring" or "data_collection"?   

That way we can put other templates not related in other subdirs.   What do you think?

**Comment by chaitanyaenr** [comment]
@mffiedler makes sense, I have moved the templates to subdirs.

**Pull request merged**